Automattic/wp-calypso
# Gutenlypso: Keep the edit-post plugin registered

See: https://github.com/Automattic/wp-calypso/issues/28424#issuecomment-443187057

> There is indeed an error in how I've modified the `ToolsMoreMenuGroup` slot logic:
>
> ```diff
> - 	{ ( fills ) => ! isEmpty( fills ) && (
> -		<MenuGroup label={ __( 'Tools' ) }>
> -			{ fills }
> -		</MenuGroup>
> -	) }
> +	<Slot fillProps={ fillProps }>
> +		{ fills => ! isEmpty( fills ) && { fills } }
> +	</Slot>
> ```
> 
> Basically I've left the brackets around `{ fills }`, so it incorrectly tries to destructure.
> 
> The reason why I've unregistered the `edit-post` plugin is because it's used to [add the Tools section to the More menu](https://github.com/WordPress/gutenberg/blob/2a9d9a4a7823d5bb27e4088a539318a162b222f5/packages/edit-post/src/plugins/index.js), and I've rewritten it from scratch.


#### Changes proposed in this Pull Request

* Keep the `edit-post` plugin registered.
* Prevent using it to fill the `ToolsMoreMenuGroup` in the More menu.

#### Testing instructions



* Open Gutenlypso `/gutenberg`.
* Open the More menu (ellipsis icon in the top right corner).
* Make sure the editor doesn't crash, and the Tools section doesn't have any duplicate items.
